ssh/server.go

```diff
@@ -434,14 +421,23 @@ var (
 	ErrNoAuth = errors.New("ssh: no auth passed yet")
 )
 
+func (s *connection) SendAuthBanner(msg string) error {
+	if s.mux != nil {
+		panic("ssh: SendAuthBanner called after handshake")
```

Comment:
why's this a panic rather than en error?

Reply:
i thought exploding would be better than an error, but no preference. Made it an error.